framework-plugins/lisk-framework-forger-plugin/src/types.ts

```diff
@@ -14,7 +14,8 @@
 
 export interface Options {
 	version: string;
-	readonly dataPath: string;
+	readonly rootPath: string;
+	readonly dataPath?: string;
```

Comment:
should it just use the same variable `dataPath`?
Also, rootPath variable name is confusing since it is used in different way in the framework